app/src/main/res/values/strings.xml

```diff
@@ -134,6 +134,9 @@
 
     <string name="pref_title_vibrate">Vibrate</string>
 
+    <string name="str_perm_title">Permission Needed</string>
+    <string name="str_perm_reason">Contacts access is needed for this feature</string>
```

Comment:
we could have this named for "contacts permission". we would be following similar steps for internet.

e.g; str_permission_contacts_reason